src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -19,46 +19,65 @@ internal partial class CertModule : CertInterface
         private static volatile X509Store s_myMachineCertStoreEx;
 
         #region internal Methods
-        internal override SslPolicyErrors VerifyRemoteCertName(X509Chain chain, bool isServer, string hostName)
+        internal override SslPolicyErrors VerifyCertificateProperties(X509Chain chain, X509Certificate2 certificate, bool checkCertName, bool isServer, string hostName)
         {
             SslPolicyErrors sslPolicyErrors = SslPolicyErrors.None;
 
-            unsafe
+            if (!chain.Build(certificate)       // Build failed on handle or on policy.
+                && chain.SafeHandle.DangerousGetHandle() == IntPtr.Zero)   // Build failed to generate a valid handle.
```

Comment:
(After getting to the end I see this is just the same logic moved from elsewhere.)